Bind commit-pinned Cargo Git inputs to checksum-verified source archives supplied
by the EasyBuild recipe so the native extension can be built without network
access.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -153,10 +153,10 @@
 [patch.crates-io]
 # packed_simd_2 = { git = "https://github.com/rust-lang/packed_simd", rev = "e57c7ba11386147e6d2cbad7c88f376aab4bdc86" }
 # simd-json = { git = "https://github.com/ritchie46/simd-json", branch = "alignment" }
-object_store = { git = "https://github.com/kdn36/arrow-rs-object-store", branch = "feat_with_dns_resolver" }
-color-backtrace = { git = "https://github.com/orlp/color-backtrace", rev = "bb62ccf1e9eb1f6b7af5f16acff1fd7151a876dd" }
+object_store = { path = "../arrow-rs-object-store-f50a6e5c564b2b5933eca15cd20ff9b5614374a1" }
+color-backtrace = { path = "../color-backtrace-bb62ccf1e9eb1f6b7af5f16acff1fd7151a876dd" }
 
 # https://github.com/tikv/jemallocator/pull/168
-tikv-jemalloc-sys = { git = "https://github.com/pola-rs/jemallocator", rev = "0d683dfb157097e2075d5e0eaf25f71f514a7552" }
+tikv-jemalloc-sys = { path = "../jemallocator-0d683dfb157097e2075d5e0eaf25f71f514a7552/jemalloc-sys" }
 
 [profile.mindebug-dev]
 inherits = "dev"
